src/API/Status/PreviewCard.vala

```diff
@@ -121,6 +129,47 @@ public class Tuba.API.PreviewCard : Entity, Widgetizable {
 		}
 	}
 
+	#if CLAPPER_0_10
+		private bool enhancer_check (string scheme, string? t_host = null) {
+			if (!Clapper.WITH_ENHANCERS_LOADER) return false;
+			bool supported = false;
+
+			string? host = t_host;
+			if (host != null && host.has_prefix ("www.")) host = host.substring (4);
+
+			var proxies = Clapper.get_global_enhancer_proxies ();
+			for (var i = 0; i < proxies.get_n_proxies (); i++) {
+				var proxy = proxies.get_proxy (i);
```

Comment:
I whitelisted "peek_proxy" for Vala in latest git, since it (Vala) understands the difference just fine.

Now its:

```vala
public Clapper.EnhancerProxy? get_proxy (uint index);
public unowned Clapper.EnhancerProxy? peek_proxy (uint index);
```

Not much, but it can save you unnecessary ref-unref cycle in this one case here (when iterating global list).